src/inc/msquic.h

```diff
@@ -496,30 +525,27 @@ typedef struct QUIC_SETTINGS {
             uint64_t KeepAliveIntervalMs                    : 1;
             uint64_t PeerBidiStreamCount                    : 1;
             uint64_t PeerUnidiStreamCount                   : 1;
-            uint64_t RetryMemoryLimit                       : 1;
-            uint64_t LoadBalancingMode                      : 1;
             uint64_t MaxOperationsPerDrain                  : 1;
             uint64_t SendBufferingEnabled                   : 1;
             uint64_t PacingEnabled                          : 1;
             uint64_t MigrationEnabled                       : 1;
             uint64_t DatagramReceiveEnabled                 : 1;
             uint64_t ServerResumptionLevel                  : 1;
-            uint64_t DesiredVersionsList                    : 1;
-            uint64_t VersionNegotiationExtEnabled           : 1;
             uint64_t MinimumMtu                             : 1;
             uint64_t MaximumMtu                             : 1;
             uint64_t MtuDiscoverySearchCompleteTimeoutUs    : 1;
             uint64_t MtuDiscoveryMissingProbeCount          : 1;
             uint64_t MaxBindingStatelessOperations          : 1;
             uint64_t StatelessOperationExpirationMs         : 1;
             uint64_t CongestionControlAlgorithm             : 1;
-            uint64_t RESERVED                               : 29;
+            uint64_t RESERVED                               : 33;
         } IsSet;
     };
 
     uint64_t MaxBytesPerKey;
     uint64_t HandshakeIdleTimeoutMs;
     uint64_t IdleTimeoutMs;
+    uint64_t MtuDiscoverySearchCompleteTimeoutUs;
```

Comment:
Can we keep the `IsSet` flags in the same order?

Reply:
Done.